client/src/routes/modals/CompletedSessionModal/CompletedSessionModal.tsx

```diff
@@ -86,9 +87,16 @@ const CompletedSessionModal = () => {
   const {t} = useTranslation('Modal.CompletedSession');
   const {payload, timestamp} = completedSessionEvent;
   const user = useUser();
+  const askForRating = useRating();
 
   const sessionTime = useMemo(() => dayjs(timestamp), [timestamp]);
 
+  useEffect(() => {
+    if (askForRating(completedSessionEvent.payload.id)) {
+      navigate('RatingModal');
+    }
+  }, [askForRating, navigate, completedSessionEvent.payload.id]);
```

Comment:
Why are we asking about this reactively here instead of when submitting feedback?